src/views/Votes.vue

```diff
@@ -125,128 +125,155 @@ import AppToolbarCentered from '@/components/AppToolbarCentered'
 import Pagination from '@/components/Pagination'
 import { explorerUriToOnion } from '@/lib/uri'
 import DelegatesTable from '@/components/DelegatesTable/DelegatesTable'
+import { computed, onMounted, ref, reactive } from "vue"
+import { useStore } from "vuex"
+import { useI18n } from 'vue-i18n'
 
 export default {
```

Comment:
defineComponent

Reply:
Alright, so I should use defineComponent function for both views and components?

Reply:
Yes.

`defineComponent` is just a type helper for TypeScript. It doesn't do anything.